static/map.js

```diff
@@ -42,6 +42,7 @@ function pad(num, size) {
 document.getElementById('pokemon-checkbox').checked = getFromStorage("displayPokemons", "true");
 document.getElementById('gyms-checkbox').checked = getFromStorage("displayGyms", "true");
 document.getElementById('coverage-checkbox').checked = getFromStorage("displayCoverage", "true");
+document.getElementById('pokestops-checkbox').checked = getFromStorage("displayPokestops", false);
```

Comment:
Well, line 45 looks fine to me Line 42, 43 and 44 should be true instead of "true".